src/features/people/PersonList/PersonTile/styled.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.article`
+  background: ${({ theme }) => theme.colors.white};
+  min-height: 339px;
+  display: grid;
+  grid-template-rows: auto 1fr;
+  box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.02);
+  };
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    min-height: 245px;
+  }
+`;
+
+export const PersonImage = styled.img`
+  padding: 10px 16px;
+  height: 286px;
+  border-radius: 5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    height: 196px;
+    padding: 8px;
+  }
+`;
+
+export const PersonFullName = styled.p`
+  margin: 0;
```

Comment:
Space below longer actor names is too small - it sholud be 22px and 15px on mobiles ;)

Reply:
by the space between longer names, do you mean the line-height? changed it to 1.3 as per project in Figma

Reply:
I mean space between the bottom edge of Tile and a bottom of name :D

Reply:
![image](https://user-images.githubusercontent.com/87578610/158158315-91c63984-ae52-4097-9a89-5711fd661c73.png)